website/content/guide/http_server.md

```diff
@@ -0,0 +1,125 @@
++++
+title = "Starting HTTP(S) server with Echo"
+description = "Serving Echo from HTTP(S) server"
+[menu.main]
+name = "HTTP(s) Server"
+parent = "guide"
++++
+
+Echo provides following convenience methods to start HTTP server with Echo as a request handler:
+
+* `Echo.Start(address string)`
+* `Echo.StartTLS(address string, certFile, keyFile interface{})`
+* `Echo.StartAutoTLS(address string)`
+* `Echo.StartH2CServer(address string, h2s *http2.Server)`
+* `Echo.StartServer(s *http.Server)`
+
+## HTTP Server
+
+`Echo.Start` is convenience method that starts http server with Echo serving requests on port 8080.
```

Comment:
The "port" part could be misleading, it sounds like "if you don't provide any port it will use 8080 as default. Also, Echo implementation will default to port 80 if the port is not provided.
See http.Server.Addr stdlib

>field Addr string
(http.Server).Addr on pkg.go.dev
Addr optionally specifies the TCP address for the server to listen on, in the form "host:port". If empty, ":http" (port 80) is used. The service names are defined in RFC 6335 and assigned by IANA. See net.Dial for details of the address format.